### ISSUE TITLE
Unclear documentation

### ISSUE THREAD

**Issue by CrisHale** [hsgw/plaid#20]
In the file https://github.com/hsgw/plaid/blob/master/doc/en/bootloader.md it says "If you buy kit, already burn bootloader in ATMEGA328p" but it's not clear what this means, or what you should do if you bought a kit. Also, I'm unsure how much of this applies to the Plaid Gold, which is what I have. Some if the documentation doesn't apply, I know, but it's not clear which does and doesn't.

There's also the line "When you burn bootloader to raw ATMEGA328p, don't forget write fuse bits." but it doesn't explain what's involved in writing the fuse bits, or how you can find out.

The section for what to do with Windows is also a bit simple. It doesn't give you any clues on how to use Zadig. The software suggests the USB item should be plugged in and chosen from a list, but in the Plaid documentation you're not asked to plug in the USB cable until the next step (Enter bootloader mode).

**Comment by BenRoe** [comment]
@CrisHale the plaid and plaid gold bootloader and firmware are identical.
Here you can find more information about how to flash the bootloader with avrdudess https://github.com/baerwolf/USBaspLoader/issues/19

**Comment by CrisHale** [comment]
@BenRoe Thanks, I will try restoring the correct bootloader on the chip I have (I'll close the issue after I give it a try, if that's okay).

**Comment by BenRoe** [comment]
@CrisHale do you changed the bootloader, or bricked the mcu?
The Plaid Gold kit comes with a pre flashed bootloader and firmware. So everything should work right after correct soldering.

**Comment by CrisHale** [comment]
@BenRoe I don't know if you remember me, but I'm the guy who soldered the USB connector on upside-down. I flashed a new bootloader before I realised that was the real problem. I assume the chip still works, but I can't test it until the replacement kit you've sent gets here and I solder it up (properly this time).
I was just thinking that if I ever do manage to fix the soldering, I could try the PCB I have again to see if it works.
